serverless.ts

```diff
@@ -0,0 +1,50 @@
+const serverlessConfiguration = {
+    service: "product-service",
+    provider: {
+      name: 'aws',
+      runtime: 'nodejs20.x',
+      stage: 'dev',
+      region: 'eu-west-1',
+    },
+    plugins: [
+      'serverless-offline'
+  ],
+    functions: {
+        getProductsList: {
+          handler: '.webpack/getProductsList.getProductsList',
```

Comment:
Not webpack, we should specified src/handlers/...

Reply:
You need to setup the following plugin: https://www.serverless.com/plugins/serverless-webpack